src/openforms/scss/components/admin/_logic-evaluated-table.scss

```diff
@@ -0,0 +1,19 @@
+.logic-rules-tables {
+    width: 100%;
+}
+
+.property-name-width {
+    width: 70%;
+}
+
+.rule-width {
+    width: 60%;
+}
+
+.trigger-width {
+    width: 10%;
+}
+
+.rule-list {
+    list-style: none;
+}
```

Comment:
We tend to stick with http://getbem.com/ for component structure, so after the backend feedback has processed, we should walk through that in more detail :-)